src/store/modules/app/actions.js

```diff
@@ -56,47 +56,43 @@ const actions = {
    *  @param {string} keyboard new keyboard we are switching to
    *  @return {object} promise that will be fulfilled once action is complete
    */
-  changeKeyboard({ state, commit, dispatch }, keyboard) {
+  async changeKeyboard({ state, commit, dispatch }, keyboard) {
```

Comment:
just make this a synchronous function

Reply:
I'm confused, why does the code say "async"  but your comment says the opposite?

Reply:
It's the irony of async. It makes async things appear synchronous to the developer. Hence air quotes.